packages/code-studio/src/dashboard/linker/Linker.tsx

```diff
@@ -41,10 +41,10 @@ export type PanelProps = {
 
 export type Panel = Component<PanelProps>;
 
-export type LinkFilterMapValue = {
+export type LinkFilterMapValue<T = unknown> = {
   columnType: string;
   text: string;
-  value: unknown;
+  value: T;
 };
 
 export type LinkFilterMap = Map<string, LinkFilterMapValue>;
```

Comment:
I think with the generic we might want to add it here as well with a default.
```suggestion
export type LinkFilterMap<T = unknown> = Map<string, LinkFilterMapValue<T>>;
```